src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -55,6 +56,7 @@ static ucs_status_t uct_cuda_copy_md_query(uct_md_h md, uct_md_attr_t *md_attr)
                                     UCS_BIT(UCS_MEMORY_TYPE_CUDA_MANAGED);
     md_attr->cap.detect_mem_types = UCS_BIT(UCS_MEMORY_TYPE_CUDA) |
                                     UCS_BIT(UCS_MEMORY_TYPE_CUDA_MANAGED);
+    md_attr->cap.dmabuf_mem_types = UCS_BIT(UCS_MEMORY_TYPE_CUDA);
```

Comment:
@yosefe dmabuf support through device query isn't provided as of 11.7, so setting to 0 for now. Let's change this when the support becomes public.

Reply:
@yosefe based on offline discussion, I've added logic to get back dmabuf_fd. I've left the above code unchanged so cuda_md will always claim it can provide dmabuf fd for pinned cuda memory but when the md is actually queried, based on memory device capabilities dmabuf_fd and offset may be set to invalid. Let me know if that suffices.